Ryujinx.HLE/HOS/Services/Time/StaticService/ISystemClock.cs

```diff
@@ -1,40 +1,41 @@
 using Ryujinx.Common;
+using Ryujinx.HLE.HOS.Ipc;
+using Ryujinx.HLE.HOS.Kernel.Common;
+using Ryujinx.HLE.HOS.Kernel.Threading;
 using Ryujinx.HLE.HOS.Services.Time.Clock;
+using System;
 
 namespace Ryujinx.HLE.HOS.Services.Time.StaticService
 {
     class ISystemClock : IpcService
     {
         private SystemClockCore _clockCore;
         private bool            _writePermission;
+        private bool           _bypassUninitializedClock;
```

Comment:
```suggestion
        private bool            _bypassUninitializedClock;
```
and same for the next `_operationEventReadableHandle` var.